src/drivers/pwm_esc/pwm_esc.cpp

```diff
@@ -31,9 +31,9 @@
  *
  ****************************************************************************/
 
-/**
+/**const uint32_t reversible_outputs = _mixing_output.reversibleOutputs();
```

Comment:
leftovers inside comment?

Reply:
ah, that's dropped there by mistake. I will clean up